### PR TITLE
RSS feed and API changes

### PR DESCRIPTION
**Pull request by juliantoledo**

### CODE REVIEW

**Comment by juliantoledo** [comment]
@andreban where is the comment? I cannot find it.

File: controllers/api/pwa.js [hunk 1]
```diff
@@ -128,34 +160,48 @@ const rssWriter = new RssWriter();
...
-        }
-        case 'rss': {
-          rssWriter.write(res, result.pwas);
-          break;
-        }
-        default: {
-          jsonWriter.write(res, result.pwas);
-        }
+
+  return new Promise((resolve, reject) => {
```
Comment (andreban) [inline controllers/api/pwa.js]: We could simplify this with something like:

```js
let promise;
if (req.params.id) {
  promise = findPwa(req.params.id);
} else {
  promise = pwaLib.list(...)
}
promise
  .then(result) => {
    ...
  }

```

**Comment by andreban** [comment]
Failed to save it, sorry. Re-added.

File: controllers/api/pwa.js (hunk 1 above)
Comment (juliantoledo) [reply controllers/api/pwa.js]: Good catch! fixed.

**Pull request merged**